lib/product.rb

```diff
@@ -0,0 +1,19 @@
+class Product
```

Comment:
It looks like the readme asked for `hoard` method and `is_hoarded` instance variable on your product, try and look at the interaction pattern and the readme a bit closer next time :)